From 90dc0cd88cf76ccdad1b74880b04c8e029043dcb Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Wed, 10 Mar 2021 09:25:40 +0100
Subject: [PATCH 1761/1851] crypto: aes: zynqmp: Remove carriage return from
 one print

commit b05496d4fe9163455bd9df4b5dd95427c2b39062 from
https://github.com/Xilinx/linux-xlnx.git

\r is causing one more empty line in dmesg log like this.

[    3.082830] zynqmp_aes zynqmp_aes: AES Successfully Registered
[    3.082830]
[    3.090050] alg: No test for xilinx-keccak-384 (zynqmp-keccak-384)

That's why remove to avoid this issue.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/crypto/zynqmp-aes.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/zynqmp-aes.c b/drivers/crypto/zynqmp-aes.c
index 43e6610dd0a7..c507a7d0c5f6 100644
--- a/drivers/crypto/zynqmp-aes.c
+++ b/drivers/crypto/zynqmp-aes.c
@@ -297,7 +297,7 @@ static int zynqmp_aes_probe(struct platform_device *pdev)
 	if (ret)
 		goto err_algs;
 
-	dev_info(dev, "AES Successfully Registered\n\r");
+	dev_info(dev, "AES Successfully Registered\n");
 	return 0;
 
 err_algs:
-- 
2.31.1

